### PR TITLE
Tweaks.

### PR DESCRIPTION
**Pull request by digimaun**

### CODE REVIEW

**Review by c-ryan-k — APPROVED** [review]

File: clients/dotnet/Microsoft.IoT.ModelsRepository.CommandLine/readme.md [hunk 1]
```diff
@@ -9,27 +9,17 @@ This solution includes a command line project `Microsoft.IoT.ModelsRepository.Co
 
 ### Install the dmr-client command line tool
 
-The tool is distributed as source code and requires `dotnet sdk 3.1` or `dotnet sdk 5.0` to build and install.
+The repository command line tool is published on [NuGet](https://www.nuget.org/packages/Microsoft.IoT.ModelsRepository.CommandLine) and requires `dotnet sdk 3.1` or `dotnet sdk 5.0`.
```
Comment (rido-min) [inline clients/dotnet/Microsoft.IoT.ModelsRepository.CommandLine/readme.md:12]: should we add a note about .NET 6 not being supported?

Comment (rido-min) [inline clients/dotnet/Microsoft.IoT.ModelsRepository.CommandLine/readme.md:12]: The repository command line tool -> The Device Models Repository command line tool, aka `dmr-client`

File: clients/dotnet/Microsoft.IoT.ModelsRepository.CommandLine/readme.md [hunk 2]
```diff
@@ -9,27 +9,17 @@ This solution includes a command line project `Microsoft.IoT.ModelsRepository.Co
...
 
-> If you have a prior version of the tool installed, be sure to uninstall before running the below install scripts.
+You can use the `dotnet` command line via the  `dotnet tool install` command to install `dmr-client`. The following is an example to install `dmr-client` as a global tool:
```
Comment (rido-min) [inline clients/dotnet/Microsoft.IoT.ModelsRepository.CommandLine/readme.md:14]: Does the tool support no-global installs ?

Comment (digimaun) [reply clients/dotnet/Microsoft.IoT.ModelsRepository.CommandLine/readme.md:14]: Yes, but its not as simple of an experience as global installs.

The user needs to create a new tool manifest file in the location of their choice using `dotnet new tool-manifest`, then in the same location run `dotnet tool install Microsoft.IoT.ModelsRepository.CommandLine --version 1.0.0-beta.4 [--add-source]`.

After the local tool install, you would run it via `dotnet tool run dmr-client`.

Comment (rido-min) [reply clients/dotnet/Microsoft.IoT.ModelsRepository.CommandLine/readme.md:14]: got it, maybe adding a link to explain how to perform non-global installs?

Comment (digimaun) [reply clients/dotnet/Microsoft.IoT.ModelsRepository.CommandLine/readme.md:14]: will do

**Pull request merged**